IoTBlockly/ChakraHost/ChakraHostAsync.cs

```diff
@@ -22,20 +23,24 @@ public class ChakraHostAsync
     {
         private JavaScriptSourceContext currentSourceContext = JavaScriptSourceContext.FromIntPtr(IntPtr.Zero);
         private JavaScriptRuntime runtime;
-        private Queue taskQueue = new Queue();
-        private IAsyncAction workItem;
-        public bool running { get; private set; }
+       // private Queue taskQueue = new Queue();
```

Comment:
Uhm... I guess it's ok to remove this as we don't use the continuations... but why remove this? did they create problems?